angular-ui-router/angular-ui-router.d.ts

```diff
@@ -7,13 +7,8 @@
 
 // Support for AMD require and CommonJS
 declare module 'angular-ui-router' {
-    // Since angular-ui-router adds providers for a bunch of
-    // injectable dependencies, it doesn't really return any
-    // actual data except the plain string 'ui.router'.
-    //
-    // As such, I don't think anybody will ever use the actual
-    // default value of the module.  So I've only included the
-    // the types. (@xogeny)
+    export default 'ui.router';
```

Comment:
is this right? https://github.com/angular-ui/ui-router/blob/master/release/angular-ui-router.js#L9